Homework2/PreprocessingScripts/Exp1/Test/proc.sub_03

```diff
@@ -0,0 +1,507 @@
+# script setup
+
+# take note of the AFNI version
+afni -ver
+
+# check that the current AFNI version is recent enough
+afni_history -check_date 23 Sep 2016
+if ( $status ) then
+    echo "** this script requires newer AFNI binaries (than 23 Sep 2016)"
+    echo "   (consider: @update.afni.binaries -defaults)"
+    exit
+endif
+
+# the user may specify a single subject to run with
+if ( $#argv > 0 ) then
+    set subj = $argv[1]
+else
+    set subj = sub_03
+endif
+
+# assign output directory name
+set output_dir = $subj.results
+
+# verify that the results directory does not yet exist
+if ( -d $output_dir ) then
+    echo output dir "$subj.results" already exists
+    exit
+endif
+
+# set list of runs
+set runs = (`count -digits 2 1 1`)
+
+# create results and stimuli directories
+mkdir $output_dir
+mkdir $output_dir/stimuli
+
+# copy stim files into stimulus directory
+cp /Users/Kayleigh/Dropbox/IBRAIN/Homework2/timing_DNR.1D     \
+    /Users/Kayleigh/Dropbox/IBRAIN/Homework2/timing_DResp.1D  \
+    /Users/Kayleigh/Dropbox/IBRAIN/Homework2/timing_LCor.1D   \
+    /Users/Kayleigh/Dropbox/IBRAIN/Homework2/timing_LIncor.1D \
+    /Users/Kayleigh/Dropbox/IBRAIN/Homework2/timing_LNR.1D $output_dir/stimuli
+
+# copy anatomy to results dir
+3dcopy                                                                                       \
+    /Users/Kayleigh/Dropbox/IBRAIN/Homework2/sub-03/ses-test/anat/sub-03_ses-test_T1w.nii.gz \
+    $output_dir/sub-03_ses-test_T1w
+
+# ============================ auto block: tcat ============================
+# apply 3dTcat to copy input dsets to results dir, while
+# removing the first 0 TRs
+3dTcat -prefix $output_dir/pb00.$subj.r01.tcat \
+    /Users/Kayleigh/Dropbox/IBRAIN/Homework2/sub-03/ses-test/func/sub-03_ses-test_task-linebisection_bold.nii.gz'[0..$]'
+
+# and make note of repetitions (TRs) per run
+set tr_counts = ( 238 )
+
+# -------------------------------------------------------
+# enter the results directory (can begin processing data)
+cd $output_dir
+
+
+# ========================== auto block: outcount ==========================
+# data check: compute outlier fraction for each volume
+touch out.pre_ss_warn.txt
+foreach run ( $runs )
+    3dToutcount -automask -fraction -polort 4 -legendre                     \
+                pb00.$subj.r$run.tcat+orig > outcount.r$run.1D
+
+    # outliers at TR 0 might suggest pre-steady state TRs
+    if ( `1deval -a outcount.r$run.1D"{0}" -expr "step(a-0.4)"` ) then
+        echo "** TR #0 outliers: possible pre-steady state TRs in run $run" \
+            >> out.pre_ss_warn.txt
+    endif
+end
+
+# catenate outlier counts into a single time series
+cat outcount.r*.1D > outcount_rall.1D
+
+# ================================= tshift =================================
+# time shift data so all slice timing is the same 
+foreach run ( $runs )
+    3dTshift -tzero 0 -quintic -prefix pb01.$subj.r$run.tshift \
```

Comment:
Remember that the slice timing information is not stored correctly in these files, so you need to add 

```-tpattern alt+z``` 

here. Note the warning in the log file:

```
[7m*+ WARNING:[0m dataset is already aligned in time!
[7m*+ WARNING:[0m ==>> output dataset is just a copy of input dataset
```